configuration/initializer_config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "models" : [
+     
+        {
+            "model_name": "syncable.units.mapping",
+            "folder": "syncable_units_mapping",
+            "field_rules": {}
+        },
+        {
+            "model_name": "product.template",
```

Comment:
Is this to load the drugs?
If so, the default Drug Loader now supports loading costs and uoms

Reply:
Removed